### PR TITLE
[269] Update Files Versions API documentation

### PR DESCRIPTION
**Pull request by settermjd**
This PR updates the Files Versions API documentation to make it more comprehensive, better formatted, and easier and more efficient to read.

### CODE REVIEW

File: modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <d:prop>
+        <d:resourcetype>
+          <d:collection/>
+        </d:resourcetype>
+      </d:prop>
+      <d:status>HTTP/1.1 200 OK</d:status>
+    </d:propstat>
+  </d:response>
+  <d:response>
+    <d:href>/remote.php/dav/meta/43/v/1576663297</d:href>
```
Comment (PVince81) [inline modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml]: might be nice to add at least a second file, to illustrate that there can be more than one ?

File: modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <d:getcontentlength>163</d:getcontentlength>
+        <d:resourcetype/>
+        <d:getetag>"9cbeacc4d9ded5801341624599507048"</d:getetag>
+        <d:getcontenttype>text/plain</d:getcontenttype>
+      </d:prop>
+      <d:status>HTTP/1.1 200 OK</d:status>
+    </d:propstat>
+    <d:propstat>
+      <d:prop>
+        <d:quota-used-bytes/>
```
Comment (PVince81) [inline modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml]: I think you should simply not query these properties as they are not useful for versions

**Comment by settermjd** [comment]
Thanks for the feedback, @PVince81. I'll address it tomorrow.

**Comment by mmattel** [comment]
@settermjd any update on this ?

**Comment by settermjd** [comment]
I'm aiming to finalise it this week.

File: modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml (hunk 1 above)
Comment (settermjd) [reply modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml]: Did you mean "at least a second version"? I'm guessing that that's what you meant.

Comment (settermjd) [reply modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml]: I just updated the content with an example that shows a file with multiple versions.

File: modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml (hunk 2 above)
Comment (settermjd) [reply modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml]: They're not being specifically queried for. That said, I'll add a PROPFIND file to remove them.

**Comment by settermjd** [comment]
@mmattel / @PVince81 both comments have been addressed.

**Comment by mmattel** [comment]
To which branches do we need backporting?

**Comment by settermjd** [comment]
> To which branches do we need backporting?

Not sure. Will find out.

**Comment by settermjd** [comment]
From research, it seems that this can be backported to both 10.4 & 10.3.

**Pull request merged**